x-pack/plugin/security/src/test/java/org/elasticsearch/xpack/security/audit/logfile/LoggingAuditTrailFilterTests.java

```diff
@@ -127,13 +128,7 @@ public void testPolicyDoesNotMatchNullValuesInEvent() throws Exception {
         final Settings.Builder settingsBuilder = Settings.builder().put(settings);
         // filter by username
         final List<String> filteredUsernames = randomNonEmptyListOfFilteredNames();
-        final List<User> filteredUsers = filteredUsernames.stream().map(u -> {
-            if (randomBoolean()) {
-                return new User(u);
-            } else {
-                return new User(new User(u), new User(UNFILTER_MARKER + randomAlphaOfLengthBetween(1, 4)));
-            }
-        }).collect(Collectors.toList());
```

Comment:
This is an example where it is no longer necessary to randomise whether User has an inner user.